frontend/.eslintrc.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   env: {
     browser: true,
     es6: true,
+    jest: true,
```

Comment:
please rewrite it with override so it won't be true in all the components

[https://eslint.org/docs/user-guide/configuring#configuration-based-on-glob-patterns](url)